### PR TITLE
[翻译更新] internal

### PR DESCRIPTION
**Pull request by liaopeiyuan**

### CODE REVIEW

**Comment by liaopeiyuan** [comment]
Translations that I'm unsure of:

list - 表
active element - 活跃元素
instance - 实例
overhead - 额外开销

File: locales/zh_CN/LC_MESSAGES/internal.po [hunk 1]
```diff
@@ -100,49 +106,64 @@ msgstr ""
...
+"``bitmasked`` 节点们（``S2``）的表。"
 
 #: ../../internal.rst:64
 msgid ""
-"The list of ``root`` node always has exactly one element (instance), so "
-"we never clear or re-generate this list."
+"The list of ``root`` node always has exactly one element (instance), so we never "
+"clear or re-generate this list."
 msgstr ""
+"关于 ``root`` 节点的表总会恰好有一个元素（实例），所以我们永远不会去清空或者重新"
```
Comment (rexwangcc) [inline locales/zh_CN/LC_MESSAGES/internal.po]:
```suggestion
"关于 ``root`` 节点的表总会有且仅有一个元素（实例），所以我们永远不会去清空或者重新"
```
how about this?

**Review by rexwangcc — APPROVED** [review]
LGTM in general, thanks for updating the translation!!

File: locales/zh_CN/LC_MESSAGES/internal.po (hunk 1 above)
Comment (liaopeiyuan) [reply locales/zh_CN/LC_MESSAGES/internal.po]: That works.

**Comment by archibate** [comment]
Thank for the hard work!

**Pull request merged**